glean-core/android/src/main/java/mozilla/telemetry/glean/net/BaseUploader.kt

```diff
@@ -74,12 +74,6 @@ class BaseUploader(d: PingUploader) : PingUploader by d {
         }
     }
 
-    /**
-     * TEST-ONLY. Allows to set specific dates for testing.
-     */
-    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal fun getCalendarInstance(): Calendar { return Calendar.getInstance() }
```

Comment:
oh hu, guess we weren't using this and just forgot to remote it?

Reply:
Yes :-)